### PR TITLE
Add config.json for RepoSense and modify gitignore

### PR DESCRIPTION
**Pull request by Hilda-Ang**

### CODE REVIEW

File: _reposense/config.json [hunk 1]
```diff
@@ -0,0 +1,26 @@
+{
+  "authors":
+    [
+    {
+      "githubId": "Hilda-Ang",
+        "displayName": "HILDA...RAENI",
```
Comment (rongjiecomputer) [inline _reposense/config.json]: Can't we use full names instead of redacted ones?

Comment (Hilda-Ang) [reply _reposense/config.json]: They say must use from here [https://nus-cs2103-ay1819s1.github.io/cs2103-website/admin/reposenseConfigTemplates.html](url)

Comment (rongjiecomputer) [reply _reposense/config.json]: Got it, thanks for the clarification.

File: _reposense/config.json [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+    },
+    {
+      "githubId": "RomaRomama",
+      "displayName": "REZKY...PUTRA",
+      "authorNames": ["RomaRomama"]
+    },
+    {
+      "githubId": "GabrielYik",
+      "displayName": "YIK R...BRIEL",
+      "authorNames": ["GabrielYik"]
```
Comment (GabrielYik) [inline _reposense/config.json]:
```suggestion
      "authorNames": ["GabrielYik", "Gabriel Yik Ren Jun"]
```

Comment (GabrielYik) [reply _reposense/config.json]: I didn't know I was using `Gabriel Yik Ren Jun` in my commits

**Pull request merged**